src/adagrams.js

```diff
@@ -1,15 +1,133 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const numTiles = 10;
+  const lettersInHand = [];
+  const currentLetterPool = [...listOfLetters]
+  for (let i = 0; i < numTiles; i++) {
+    // 98 letters, i want index from 0 to 97 inclusive
+    let randIndex = Math.floor(Math.random() * currentLetterPool.length);
+    let randLetter = currentLetterPool[randIndex];
+    lettersInHand.push(randLetter);
+    currentLetterPool.splice(randIndex, 1)
+  }
+  return lettersInHand
 };
 
 export const usesAvailableLetters = (input, lettersInHand) => {
-  // Implement this method for wave 2
+  const letterMap = lettersInHandToDict(lettersInHand);
+  input = input.toUpperCase();
+  for (const char of input) {
```

Comment:
Can also write lines 17-18 in one line:
```suggestion
  for (const char of input.toUpperCase()) {
```